port: /dev/xtark_base        #通信端口号
odom_frame: odom         #里程计坐标系
base_frame: base_link    #底盘坐标系
publish_odom_transform: true    #是否发布odom->base_link TF变换

robot_radius: 0.226      #底盘半径
robot_wheel_diameter: 0.13999  #底盘轮径参数，用作里程计线速度相关解算
robot_wheel_track: 0.3715415       #底盘轮距参数，用作里程计旋转相关解算


enable_sonar: true   #超声波驱动碰撞躲避

safe_distance: 0.1    #机器人超声波安全距离，小于此距离，机器人将会急停
inflation_slow_distance: 0.05    #超声波减速膨胀区域

sonar0_offset_x: 0.15556
sonar0_offset_y: 0.15556
sonar0_offset_yaw: 0.7854

sonar1_offset_x: 0.22
sonar1_offset_y: 0.0
sonar1_offset_yaw: 0.0

sonar2_offset_x: 0.15556
sonar2_offset_y: -0.15556
sonar2_offset_yaw: -0.7854

sonar3_offset_x: -0.15556
sonar3_offset_y: -0.15556
sonar3_offset_yaw: -2.3562

sonar4_offset_x: -0.15556
sonar4_offset_y: 0.15556
sonar4_offset_yaw: 2.35




